_includes/tutorials/aggregating-minmax/flinksql/code/src/test/java/io/confluent/developer/FlinkSqlAggregatingMinMaxTest.java

```diff
@@ -0,0 +1,37 @@
+package io.confluent.developer;
+
+
+import org.apache.flink.table.api.TableResult;
+import org.junit.Test;
+
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+
+public class FlinkSqlAggregatingMinMaxTest extends AbstractFlinkKafkaTest {
+
+  @Test
+  public void simpleSelect() throws Exception {
+    // create base movie sales table and aggregation table, and populate with test data
+    tEnv.executeSql(getResourceFileContents("create-movie-sales.sql.template",
+        Optional.of(kafkaPort), Optional.of(schemaRegistryPort))).await();
+    tEnv.executeSql(getResourceFileContents("populate-movie-sales.sql"));
+    tEnv.executeSql(getResourceFileContents("create-movie-sales-by-year.sql.template",
+        Optional.of(kafkaPort), Optional.of(schemaRegistryPort))).await();
```

Comment:
with schema registry when you set the port to something like `bogus` it will automatically use the `MockSchemaRegistry`, but I'm not sure how that would play out with the flink environment - so I'd leave this as is for now.